backend/api/v1/issue_service.go

```diff
@@ -111,22 +122,41 @@ func (s *IssueService) ListIssues(ctx context.Context, request *v1pb.ListIssuesR
 		return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("page size must be non-negative: %d", request.PageSize))
 	}
 
-	projectID, err := common.GetProjectID(request.Parent)
+	requestProjectID, err := common.GetProjectID(request.Parent)
 	if err != nil {
 		return nil, status.Errorf(codes.InvalidArgument, err.Error())
 	}
-	var projectResourceID *string
-	if projectID != "-" {
-		project, err := s.store.GetProjectV2(ctx, &store.FindProjectMessage{
-			ResourceID: &projectID,
-		})
+
+	projectIDs, err := func() (*[]string, error) {
```

Comment:
Can we move to a separate function?
- Reduce the size of the function.
- Common function that applies to other places as well.